modules/registry-common-terms.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * registry/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-registry-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+container::
+Lightweight and executable images that consist software and all its dependencies. Because containers virtualize the operating system, you can run containers anywhere, from a data center to a public or private cloud to your local host.
+
+Image Registry Operator::
+The Image Registry Operator runs in the `openshift-image-registry` namespace, and manages the registry instance in that location.
+
+image repository::
+An image repository is a collection of related container images and tags identifying images.
+
+mirror registry::
+A registry that holds the mirror of {product-title} images.
+
+namespace::
+A namespace isolates specific system resources that are visible to all processes. Inside a namespace, only processes that are members of that namespace can see those resources.
```

Comment:
where do we use this term in the registry docs?

Reply:
A term namespace is mentioned under https://docs.openshift.com/container-platform/4.12/registry/configuring_registry_storage/configuring-registry-storage-vsphere.html

Reply:
This definition seems to be about kernel namespaces - https://en.wikipedia.org/wiki/Linux_namespaces
but we usually use `namespace` as a Kubernetes namespace - https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/ - on k8s level there are no processes

Reply:
acked